exchange/docs-conceptual/connect-to-exchange-online-powershell.md

```diff
@@ -47,136 +47,424 @@ To use the older Exchange Online Remote PowerShell Module to connect to Exchange
 
-   **This example connects to Exchange Online PowerShell in a Microsoft 365 DoD organization**:
+- If you aren't using MFA, you should be able to use the _Credential_ parameter instead of the _UserPrincipalName_ parameter. First, run the command `$Credential = Get-Credential`, enter your username and password, and then use the variable name for the _Credential_ parameter (`-Credential $Credential`). If it doesn't work, then you need to use the _UserPrincipalName_ parameter.
 
-   ```powershell
-   Connect-ExchangeOnline -UserPrincipalName julia@adatum.mil -ExchangeEnvironmentName O365USGovDoD
-   ```
+- Using the module in PowerShell 7 requires version 2.0.4 or later.
 
-   **This example connects to Exchange Online PowerShell to manage another tenant**:
+The connection examples in the following sections use modern authentication, and are incapable of using Basic authentication.
 
-   ```powershell
-   Connect-ExchangeOnline -UserPrincipalName navin@contoso.com -DelegatedOrganization adatum.onmicrosoft.com
-   ```
+### Connect to Exchange Online PowerShell with an interactive login prompt
+
+1. The following examples work in Windows PowerShell 5.1 and PowerShell 7 for accounts with or without MFA:
+
+   - **This example connects to Exchange Online PowerShell in a Microsoft 365 or Microsoft 365 GCC organization**:
+
+     ```powershell
+     Connect-ExchangeOnline -UserPrincipalName navin@contoso.com
+     ```
+
+   - **This example connects to Exchange Online PowerShell in a Microsoft GCC High organization**:
+
+     ```powershell
+     Connect-ExchangeOnline -UserPrincipalName laura@blueyonderairlines.us -ExchangeEnvironmentName O365USGovGCCHigh
+     ```
 
-3. In the sign-in window that opens, enter your password, and then click **Sign in**.
+   - **This example connects to Exchange Online PowerShell in a Microsoft 365 DoD organization**:
+
+     ```powershell
+     Connect-ExchangeOnline -UserPrincipalName julia@adatum.mil -ExchangeEnvironmentName O365USGovDoD
+     ```
+
+   - **This example connects to Exchange Online PowerShell in an Office 365 Germany organization**:
+
+     ```powershell
+     Connect-ExchangeOnline -UserPrincipalName lukas@fabrikam.de -ExchangeEnvironmentName O365GermanyCloud
+     ```
+
+2. In the sign-in window that opens, enter your password, and then click **Sign in**.
 
    ![Enter your password in the Sign in to your account window.](media/connect-exo-password-prompt.png)
 
-4. **MFA only**: A verification code is generated and delivered based on the response option that's configured for your account (for example, a text message or the Microsoft Authenticator app on your device).
+   > [!NOTE]
+   > In PowerShell 7, browser-based single sign-on (SSO) is used by default, so the sign in prompt opens in your default web browser instead of a standalone dialog.
```

Comment:
```suggestion
   > In PowerShell 7, browser-based single sign-on (SSO) is used by default, so the sign-in prompt opens in your default web browser instead of a standalone dialog.
```